drivers/can/can_mcp2515.h

```diff
@@ -13,6 +13,7 @@
 #define MCP2515_RX_CNT                   2
 #define MCP2515_TX_CNT                   3
 #define MCP2515_FRAME_LEN               13
+#define MCP2515_DLC_MAX                  8
```

Comment:
There is a CAN_MAX_DLC in can.h which I just noticed when checking the stm32 send code :-) Please use that instead, as it's part of the CAN standard (non-FD).

Reply:
Done